tests/Backend/CommonPart1/DataPreviewLimitsTest.php

```diff
@@ -175,7 +203,7 @@ private function createRandomString($length)
         $alpabet = 'abcdefghijklmnopqrstvuwxyz0123456789 ';
         $randStr = '';
         for ($i = 0; $i < $length; $i++) {
-            $randStr .=  $alpabet[rand(0, strlen($alpabet)-1)];
+            $randStr .= $alpabet[rand(0, strlen($alpabet) - 1)];
```

Comment:
jsem asi nekde uklikl nejakou formatovaci zkratku...

Reply:
CTRL+ALT+L to mačkám furt dokola :D